tck/src/main/java/org/eclipse/microprofile/config/tck/configsources/DefaultConfigSourceOrdinalTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.config.tck.configsources;
+
+import javax.inject.Inject;
+import org.eclipse.microprofile.config.Config;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.testng.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author <a href="mailto:emijiang6@googlemail.com">Emily Jiang</a>
+ */
+public class DefaultConfigSourceOrdinalTest extends Arquillian {
+
+    private @Inject Config config;
+
+    @Deployment
+    public static Archive deployment() {
+        JavaArchive testJar = ShrinkWrap
+            .create(JavaArchive.class, "DefaultConfigSourceOrdinalTest.jar")
+            .addClasses(DefaultConfigSourceOrdinalTest.class)
+            .addAsManifestResource(
+                    new StringAsset(
+                        "config_ordinal=200\n" +
+                        "customer_name=Bill\n" +
+                        "customer.hobby=Badminton"
+                        ),
+                        "microprofile-config.properties")
+            .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+            .as(JavaArchive.class);
+
+        WebArchive war = ShrinkWrap
+            .create(WebArchive.class, "DefaultConfigSourceOrdinalTest.war")
+            .addAsLibrary(testJar);
+        return war;
+    }
+
+    @BeforeClass
+    public void checkSetup() {
+       //check whether the environment variables were populated by the executor correctly
+
+        if (!"45".equals(System.getenv("config_ordinal"))) {
```

Comment:
These checks don't match the properties set in the TCK instructions (upper case). In these case, since we are doing a direct check to `System.env` the name rules don't apply unfortunately.

Reply:
I got the info wrong in the TCK instructions. I've corrected it now.